### ISSUE TITLE
What is the equivalent of measureCompilationTime in v6.x to see single aggregate time compilation took?

### ISSUE THREAD

**Issue by rkriauciukas-freesat** [TypeStrong/fork-ts-checker-webpack-plugin#537]
Before upgrading to `fork-ts-checker-webpack-plugin` v6.x, after typescript checks there was a nice total time taken output:

```
Compilation took: 281211.62 ms.
No type errors found
Version: typescript 3.8.3
Time: 281248 ms
```

What is the a way in v6.x to get the same output?
Setting `typescript.profile = true` outputs split timings, but no total:

```
Issues checking in progress...
Create Watch Compiler Host: 0.00 s
I/O Read:                   0.44 s
Parse:                      1.94 s
ResolveTypeReference:       0.38 s
ResolveModule:              2.05 s
Program:                    5.11 s
Bind:                       1.29 s
Check:                      0.84 s
transformTime:              0.33 s
Emit:                       0.43 s
Semantic Diagnostics:       1.33 s
Create Watch Program:       7.86 s
Queued Tasks:               0.00 s
No issues found.
```

Not entirely sure if that's a change on `fork-ts-checker-webpack-plugin` or `TypeScript` after upgrading from 3.x to 4.x.

Thanks!

**Comment by piotr-oles** [comment]
AFAIK you can use `logger: { infrastructure: 'console' }` to see check time

**Comment by rkriauciukas-freesat** [comment]
Ah sweet! That's what I was looking for - and somehow have not tried changing the prop. Thanks @piotr-oles !

```
Issues checking in progress...
Create Watch Compiler Host: 0.00 s
I/O Read:                   0.21 s
Parse:                      1.64 s
ResolveTypeReference:       0.16 s
ResolveModule:              0.87 s
Program:                    3.08 s
Bind:                       1.17 s
Check:                      0.91 s
transformTime:              0.26 s
Emit:                       0.37 s
Semantic Diagnostics:       1.34 s
Create Watch Program:       5.78 s
Queued Tasks:               0.00 s
No issues found.
Time: 9287 ms

```
